webapp/src/components/markdownEditorInput/markdownEditorInput.tsx

```diff
@@ -74,17 +74,9 @@ const MarkdownEditorInput = (props: Props): ReactElement => {
     }, [])
 
     useEffect(() => {
-        setTimeout(() => ref.current?.focus(), 200)
-    })
-
-    useEffect(() => {
-        let isMounted = true
-        if (isEditing && isMounted) {
+        if (isEditing) {
             setEditorState(EditorState.moveSelectionToEnd(editorState))
-        }
-
-        return () => {
-            isMounted = false
+            setTimeout(() => ref.current?.focus(), 200)
         }
     }, [isEditing])
```

Comment:
Nice @kamre! You're the man!
Oddly enough, I tested this locally and this not only solves the problem you've pointed out, but also solves #1877.

Can you just make the following changes to the `useEffect` underneath this? It should end up looking like
```JS
    useEffect(() => {
        if (isEditing) {
            setEditorState(EditorState.moveSelectionToEnd(editorState))
            setTimeout(() => ref.current?.focus(), 200)
        }
    }, [isEditing])

    useEffect(() => {
        if (initialText === '' && !isEditing) {
            setTimeout(() => {
                setEditorState(EditorState.createEmpty())
            }, 200)
        }
    }, [initialText, isEditing])
 ```
 
 I had to make those additional changes because I noticed that when I click in on a comment and type my first letter, it takes the cursor to the beginning. 
 
 Once it's made, re-request a re-review and I'll approve. Thanks!

Reply:
@hahmadia I have made the changes that you propose and it indeed helps with the cursor positioning!

But I have also noticed that when all the text is removed and `onBlur` is performed then next text editing seems to be broken.